vllm/entrypoints/openai/cli_args.py

```diff
@@ -117,7 +117,7 @@ def make_arg_parser(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
         "or JSON format. "
         "Example (old format): ``'name=path'`` "
         "Example (new format): "
-        "``{\"name\": \"name\", \"local_path\": \"path\", "
+        "``{\"name\": \"name\", \"path\": \"local_path\", "
```

Comment:
NIT: we aslo support download lora model from HF
```suggestion
        "``{\"name\": \"name\", \"path\": \"lora_path\", "
```

Reply:
👌